watertap/property_models/activated_sludge/tests/test_asm2d_reaction.py

```diff
@@ -12,13 +12,17 @@
 #################################################################################
 """
 Tests for ASM2d reaction package.
-Authors: Andrew Lee
+Authors: Andrew Lee, Alejandro Garciadiego
 
-Reference:
+References:
 
 [1] Henze, M., Gujer, W., Mino, T., Matsuo, T., Wentzel, M.C., Marais, G.v.R.,
 Van Loosdrecht, M.C.M., "Activated Sludge Model No.2D, ASM2D", 1999,
 Wat. Sci. Tech. Vol. 39, No. 1, pp. 165-182
+
+[2] Flores-Alsina X., Gernaey K.V. and Jeppsson, U. "Benchmarking biological
+nutrient removal in wastewater treatment plants: influence of mathematical model
+assumptions", 2012, Wat. Sci. Tech., Vol. 65 No. 8, pp. 1496-1505
```

Comment:
Was this the reference that helped clarify the error in the literature concerning R18 and R19?

Reply:
Yes, it is the paper that utilized the Matlab file I used to validate